GenderPayGap.WebUI/Views/ActionHubOld/Promising.cshtml

```diff
@@ -0,0 +1,187 @@
+                </a>
+                to see how to attract and hire returners. For example:
+            </p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>Target places where returners are likely to be looking</li>
+                <li>Ensure the recruitment process is returner-friendly</li>
+                <li>Offer support before and during the assessment</li>
+            </ul>
+        </li>
+        
+        <li>
+            <h3 class="govuk-heading-s">Offer mentoring and sponsorship</h3>
+            <p class="govuk-body">
+                Although quite similar roles, mentors provide guidance and advice to their mentee while sponsors support the advancement and visibility of the person they are sponsoring. Some evidence suggests that mentoring programmes work very well for some women but not for others. It is not clear based on existing evidence whether sponsorships are more effective than mentoring, or how best to run mentoring and sponsorship programmes so they are effective.
+            </p>
+        </li>
+        
+        <li>
+            <h3 class="govuk-heading-s">Offer networking programmes</h3>
+            <p class="govuk-body">
+                Some evidence suggests that formal networking programmes where members meet and share information and career advice can be helpful for some women but not others.
+                More work is needed to understand the effects of networking programmes, and whether they need to have particular features in order to be successful.
+            </p>
+        </li>
+        
+        <li>
+            <h3 class="govuk-heading-s">Set internal targets</h3>
+            <p class="govuk-body">
+                It is important to ensure employers’ equality goals are clear and realistic, and that progress towards them can be tracked. “Improving gender equality at my organisation” or “reducing my organisation’s gender pay gap” can be overarching goals, but they are not specific and they therefore risk being unsuccessful. One way of increasing the likelihood that goals will be reached is by setting specific, time-bound targets: what change will be achieved, and by when?
+            </p>
+        </li>
+    </ol>
+</div>
+
+<div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
+    <div class="govuk-accordion__section ">
+        <div class="govuk-accordion__section-header">
+            <h2 class="govuk-accordion__section-heading">
+                <span class="govuk-accordion__section-button" id="accordion-default-heading-1">
+                    References
+                </span>
+            </h2>
+        </div>
+        <div id="accordion-default-content-1" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-1">
+            <div>
+                <ol class="govuk-list govuk-list--number">
+                    <li>

```

Comment:
This new `<ol>` / `<li>` structure for the References section looks much cleaner 😄